lib/plugins/aws/package/compile/events/alb/lib/healthCheck.test.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+const { expect } = require('chai');
+const runServerless = require('../../../../../../../../tests/utils/run-serverless');
+const fixtures = require('../../../../../../../../tests/fixtures');
+
+const healthCheckDefaults = {
+  HealthCheckEnabled: false,
+  HealthCheckPath: '/',
+  HealthCheckIntervalSeconds: 35,
+  HealthCheckTimeoutSeconds: 30,
+  HealthyThresholdCount: 5,
+  UnhealthyThresholdCount: 5,
+  Matcher: { HttpCode: '200' },
+};
+
+describe('ALB TargetGroup Health Checks', () => {
+  let cfResources;
+  let naming;
+
+  it('should be forcibly reverted to its default state (disabled) if healthCheck is not set', () =>
+    fixtures
+      .extend('functionHealthCheck')
```

Comment:
It'll be cleaner and more optimal to:

1. Extend `function` fixture with all tested combinations of `alb` events (I would probably use different function for each, and we can just create different function for each case
2. Run `runServerless` once (in _before_ caluse) and then in following `it's` test the outcome for each function individually